From 794e41fcbcefa59cad41e34ce28b44430bf2db42 Mon Sep 17 00:00:00 2001
From: centny <centny@gmail.com>
Date: Sun, 18 Apr 2021 12:32:43 +0800
Subject: [PATCH] pixman-build-ios

---
 Makefile.am  | 2 +-
 configure.ac | 3 +--
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index dccbf16..6729606 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = pixman demos test
+SUBDIRS = pixman
 
 pkgconfigdir=$(libdir)/pkgconfig
 pkgconfig_DATA=pixman-1.pc
diff --git a/configure.ac b/configure.ac
index 8231e67..b82e1cf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1134,8 +1134,7 @@ AC_OUTPUT([pixman-1.pc
            Makefile
 	   pixman/Makefile
 	   pixman/pixman-version.h
-	   demos/Makefile
-	   test/Makefile])
+	   ])
 
 m4_if(m4_eval(pixman_minor % 2), [1], [
    echo
-- 
2.24.3 (Apple Git-128)

